pkg/cri/sbserver/sandbox_run.go

```diff
@@ -382,6 +382,10 @@ func cniNamespaceOpts(id string, config *runtime.PodSandboxConfig) ([]cni.Namesp
 		opts = append(opts, cni.WithCapabilityDNS(*dns))
 	}
 
+	if cgroup := config.GetLinux().GetCgroupParent(); cgroup != "" {
+		opts = append(opts, cni.WithCapability("cgroupPath", cgroup))
```

Comment:
Since it is well-known capability, I think we should also add `cni.WithCapabilityCgroupPath` helper in containerd/go-cni package.